### ISSUE TITLE
[v6r20] ParametricJobs with Lists for parameter sequence not working

### ISSUE THREAD

**Issue by andresailer** [DIRACGrid/DIRAC#3720]
Using bulksubmission in the transformation system I get problems when there is a parameter sequence that is a sequence of lists.
E.g.:
```
 'Parameters.ProductionOutputData': {'type': 'JDL',
                                     'value': ['/ilc/prod/clic/test/431.2gev/bb/CLIC_o3_v14/REC/00401136/000/bb_rec_401136_1.slcio;/ilc/prod/clic/test/431.2gev/bb/CLIC_o3_v14/DST/00401136/000/bb_dst_401136_1.slcio',
                                              '/ilc/prod/clic/test/431.2gev/bb/CLIC_o3_v14/REC/00401136/000/bb_rec_401136_2.slcio;/ilc/prod/clic/test/431.2gev/bb/CLIC_o3_v14/DST/00401136/000/bb_dst_401136_2.slcio',
....]
```
I think because of the semi-colon as the entry separation the parsing fails and later gives back
```
attribute Parameters.ProductionOutputData is a list: ['/ilc/prod/clic/test/431.2gev/bb/CLIC_o3_v14/REC/00401136/000/bb_rec_401136_1.slci']
```
And the submission fails with `Different length of parameter vectors`

Not sure if the ClassAd parsing or the list encoding or both need to be fixed.

(So far I only tested the first two steps of my Transformation chains which did not have multiple outputdata files).

**Comment by atsareg** [comment]
Did you try to do a real list of lists rather than a list of semicolon separated strings ?

**Comment by andresailer** [comment]
Yes, some parameters are list of lists:
```
Parameters.Site': {'type': 'JDL',
                     'value': [['LCG.LAPP.fr',
                                'LCG.Freiburg.de',
                                'OSG.BNL.us',
```
But this all happens inside the transformation system, so I am not sure where the ProductionOutputData values get injected into the parameters/jdl.

**Comment by andresailer** [comment]
I think it is this line:

https://github.com/DIRACGrid/DIRAC/blob/1a12b47509ebb892eb81e64686efc45b3e2a3d43/TransformationSystem/Client/TaskManager.py#L584

**Comment by andresailer** [comment]
Fixed with #3722
